account_bank_statement_import_mt940_base/tests/test_import_bank_statement.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2017 Onestein (<http://www.onestein.eu>)
```

Comment:
@feketemihai Could you also test the base module for SNS Bank MT940 files (create an extra test), just to be sure besides Rabobank & ING. See attachment for an example file. 

[tr-info_15785279_20180104123832.zip](https://github.com/OCA/bank-statement-import/files/1603485/tr-info_15785279_20180104123832.zip)

And even documentation (Dutch too bad) if needed:
[6_0016_35_1511_Informatiewijzer_MT940_rapportage.pdf](https://github.com/OCA/bank-statement-import/files/1603487/6_0016_35_1511_Informatiewijzer_MT940_rapportage.pdf)

If the test fails on some SNS specific thing, let me know I can help translating to English if needed.

Reply:
Ps. look also at the V10 PR: https://github.com/OCA/bank-statement-import/pull/125

Where I also propose to add a SNS MT940 testcase, maybe we should re-invent the wheel if not needed.

Reply:
@danger89 I've added SNS testcase, the ideea is that the module doesn't handle multiple statements in the same file, so i update the file by reducing the number of statements to the first one, so to import it you will have to export daily statements, not from a period of time...

Reply:
@danger89 i will try to add it in base at the begining of next week.